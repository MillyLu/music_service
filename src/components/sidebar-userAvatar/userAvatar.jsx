import * as Styled from './styles';


export function UserAvatar() {
    return(
        <Styled.SidebarAvatar />
    )
}