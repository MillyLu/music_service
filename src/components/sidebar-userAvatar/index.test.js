/* eslint-disable no-undef */
import { render } from "@testing-library/react";

import { UserAvatar } from "./userAvatar";


describe('<UserAvatar />', () => {
    it('should render and match snapshot', () => {
        const {container} = render(<UserAvatar />);

        expect(container).toMatchSnapshot();
    })
})