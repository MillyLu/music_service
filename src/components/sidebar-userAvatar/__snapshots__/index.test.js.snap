// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<UserAvatar /> should render and match snapshot 1`] = `
<div>
  <div
    class="sc-bcXHqe epXTIq"
  />
</div>
`;
